modules/ROOT/pages/flex-gateway-monitoring.adoc

```diff
@@ -0,0 +1,23 @@
+= Monitoring
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]s
```

Comment:
there's an extra "**s**" at the end of this line, please remove it.

Reply:
Removed